x-pack/platform/plugins/shared/onechat/public/application/components/tools/base_tools.tsx

```diff
@@ -36,7 +40,7 @@ const columns: Array<EuiBasicTableColumn<ToolDescriptor>> = [
     width: '60%',
     valign: 'top',
     render: (description: string) => {
-      return <EuiText size="s">{parseToolDescription(description)}</EuiText>;
+      return <EuiText size="s">{truncateAtNewline(description)}</EuiText>;
```

Comment:
just thinking out loud: a tool with a very long single-paragraph description might still take too much space in the layout. Perhaps we should consider sth like max height for a single item and truncate with ellipsis later on (but this requires handling edgecases on smaller screens).